### PR TITLE
Commerce 7396

### PR DESCRIPTION
**Pull request by garofalog**
https://issues.liferay.com/browse/COMMERCE-7396

### CODE REVIEW

**Comment by garofalog** [comment]
ci:test:sf

**Comment by garofalog** [comment]
ci:test:relevant

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -6271,7 +6271,7 @@ fix-the-sla-configuration-to-resume-accurate-reporting=Fix the SLA configuration
 fixed=Fixed
 fixed-amount=Fixed Amount
 fixed-price=Fixed Price
-fixed-shipping-description=Ship for a fixed price.
+fixed-shipping-description=Flat Rate Description
```
Comment (luca-pellizzon) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:6274]: @garofalog are you sure about this fix? You are rewriting a label, but it is wrong.

**Review by luca-pellizzon — CHANGES_REQUESTED** [review]
see comment

**Pull request closed without merging**